<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientEditDlg</class>
 <widget class="QDialog" name="ClientEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="clientNameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>客户名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientAddrEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>客户地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
